File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/d/ems/ems/EmsRstDTO.java
package iih.ci.ord.d.ems.ems;

import iih.ci.ord.d.ems.base.BaseCiDTO;
import xap.mw.core.data.FArrayList;
import xap.mw.coreitf.d.FBoolean;

/**
 * 医疗单返回结果对象
 * @author wangqingzhu
 *
 */
public class EmsRstDTO extends BaseCiDTO {

	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	
	/**
	 * 医疗单类型
	 * @return String
	 */
	public String getEmsDriver() {
		return ((String) getAttrVal("EmsDriver"));
	}
	/**
	 * 医疗单类型
	 * @param Id_or
	 */
	public void setEmsDriver(String EmsDriver) {
		setAttrVal("EmsDriver", EmsDriver);
	}
	
	
	
	/**
	 * 错误医嘱
	 * @return FArrayList
	 */
	public FArrayList getErrorEmsList() {
		return ((FArrayList) getAttrVal("ErrorEmsList"));
	}
	/**
	 * 错误医嘱
	 * @param ErrorEmsList
	 */
	public void setErrorEmsList(FArrayList ErrorEmsList) {
		setAttrVal("ErrorEmsList", ErrorEmsList);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/emsmain/ems/EmsRstDTO.cs
using iih.ci.iih.ci.ord.dto.emsmain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.dto.emsmain
{
    public class EmsRstDTO : BaseCiDTO
    {
       

        /// <summary>
        /// 医疗单驱动
        /// </summary>
        public String EmsDriver
        {
            get { return getAttrVal<String>("EmsDriver", null); }
            set { setAttrVal<String>("EmsDriver", value); }
        }

        public FArrayList ErrorEmsList
        {
            get { return getAttrVal<FArrayList>("ErrorEmsList", null); }
            set { setAttrVal<FArrayList>("ErrorEmsList", value); }
        }

    }
}
